test/Service/StorageAdapterPluginManagerFactoryTest.php

```diff
@@ -14,24 +14,19 @@
 use Laminas\Cache\Storage\StorageInterface;
 use Laminas\ServiceManager\ServiceLocatorInterface;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
 
 class StorageAdapterPluginManagerFactoryTest extends TestCase
 {
+    use ProphecyTrait;
+
     public function testFactoryReturnsPluginManager()
     {
         $container = $this->prophesize(ContainerInterface::class)->reveal();
         $factory = new StorageAdapterPluginManagerFactory();
 
         $adapters = $factory($container, AdapterPluginManager::class);
         $this->assertInstanceOf(AdapterPluginManager::class, $adapters);
-
-        if (method_exists($adapters, 'configure')) {
```

Comment:
The code here additionally tests whether $factory sets the container to the AdapterPluginManager - but do you think this is necessary? Since the name of the test is testFactoryReturnsPluginManager, I think it's okay to remove this assertion. 

Otherwise I would need to access the protected property to reimplement the previous assertion.